setup.py

```diff
@@ -17,6 +17,15 @@
 from distutils.version import LooseVersion
 from setuptools import setup, Command, find_packages
 
+# Taken from https://github.com/jupyterhub/traefik-proxy/
+# commit: 0f16dc307b72e613e71067b6498f82728461434a
+#
+# ensure cwd is on sys.path
+# workaround bug in pip 19.0
+here = os.path.dirname(__file__)
+if here not in sys.path:
+    sys.path.insert(0, here)
```

Comment:
Why is this needed? 
For versioneer?

Reply:
Yes, added a comment

Reply:
BTW, this workaround might not be needed any more, as it should be fixed in setuptools/pip (https://github.com/pypa/pip/pull/6212)

Reply:
I would keep this in here for a bit as the pip release for that was not done. (From another bug report it seems like it will be in the release coming out this weekend but I won't count on that).

Reply:
New Pip/Setuptools to fix this have been released in the meantime.

Reply:
Removed this again